kubernetes/loculus/values.yaml

```diff
@@ -6,7 +6,7 @@ disableWebsite: false
 disableBackend: false
 disablePreprocessing: false
 disableIngest: false
-disableEnaSubmission: true
+disableEnaSubmission: false
```

Comment:
For final PR it would be nice to do this in the `values_preview_server.yaml` rather than in the default Helm chart - until ENA submission is fully mature

Reply:
(and actually even after that, because many Loculus instances wouldn't use this)

Reply:
added in https://github.com/loculus-project/loculus/issues/2831